build_tools/bazel/iree_bytecode_module.bzl

```diff
@@ -15,7 +15,7 @@ def iree_bytecode_module(
         src,
         module = None,
         flags = ["-iree-mlir-to-vm-bytecode-module"],
-        translate_tool = "//iree/tools:iree-translate",
+        translate_tool = "//iree/tools:iree-compile",
```

Comment:
as a follow-up we could rename this parameter to "compile_tool" (across each Bazel and CMake function)

Reply:
Added todo.